tests/Fixtures/TestBundle/Entity/DummyTour.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity;
+
+use ApiPlatform\Core\Annotation\ApiProperty;
+use ApiPlatform\Core\Annotation\ApiResource;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Dummy Tour.
+ * https://github.com/api-platform/core/issues/1107.
+ *
+ * @author Antoine Bluchet <soyuka@gmail.com>
+ *
+ * @ApiResource
+ * @ORM\Entity
+ */
+class DummyTour
```

Comment:
Or we might want to take a page out of how Doctrine keeps test cases. For example: https://github.com/doctrine/doctrine2/tree/v2.5.6/tests/Doctrine/Tests/Models/DDC3699